cli/cmd/devnet.go

```diff
@@ -172,7 +170,6 @@ func devnetDefinition(ctx context.Context) (app.Definition, error) {
 
 	defCfg := app.DefaultDefinitionConfig(ctx)
 	defCfg.ManifestFile = manifestFile
-	defCfg.OmniImgTag = buildinfo.Version()
```

Comment:
I removed this line as the devnet would not start with `"main"` as a value here either. Removing this avoided that. I'd appreciate a second look here

Reply:
This would be if building from source no, but should be fine in a release?

Reply:
It should be fine in a release I though too. But then `"main"` should have worked here too as that would mimic the same behaviour as when using the buildinfo tag version here.

Reply:
I'll use this line again, will test before publishing release on GH